1.18/Common/src/main/resources/assets/universalenchants/lang/zh_cn.json

```diff
@@ -0,0 +1,4 @@
+{
+  "commands.enchant.remove.success.single": "已移除%s附魔于%s的物品上",
```

Comment:
```suggestion
  "commands.enchant.remove.success.single": "已移除%1$s物品上的%2$s附魔",
```

Reply:
i tried it ingame, and this method seems not working
![image](https://github.com/Fuzss/universalenchants/assets/99306685/80430f37-557e-4290-aec5-3477b4a727de)

Reply:
oh,sorry,I make a typo

Reply:
I've corrected it

Reply:
But I'm not sure what the numbers mean and whether they represent counting

Reply:
i'm using offline login cuz mircrosoft's authorization sucks, and 23 is the player name